doc/creating_a_new_project.md

```diff
@@ -0,0 +1,50 @@
+# Creating a New Project
+
+## Creating a new WinRT Component DLL and referencing it in another project
+
+When creating a new DLL, it was really helpful to reference an existing DLL's `.vcxproj` like `TerminalControl.vcxproj`. While you should mostly try to copy what the existing `.vcxproj` has, here's a handful of things to double check for as you go along.
+
+- [ ] Make sure to `<Import>` our pre and post build props in your `NewDLL.vcxproj`
+```
+<!-- pre props --> 
+<Import Project="..\..\..\common.openconsole.props" Condition="'$(OpenConsoleDir)'==''" />
+<Import Project="$(OpenConsoleDir)src\cppwinrt.build.pre.props" />
+<!-- post props -->

```

Comment:
Might make it a bit more obvious that all the pres go at the top of the file and the posts go at the bottom. With them all sitting together here, someone could get mistaken.